app/forms/declaration/TotalPackageQuantity.scala

```diff
@@ -30,17 +30,24 @@ object TotalPackageQuantity extends DeclarationPage {
 
   val formId = "TotalPackageQuantity"
 
+  def applyRequired(value: String): TotalPackageQuantity = TotalPackageQuantity(if (value.isEmpty) None else Some(value))
+
+  def unapplyRequired(value: TotalPackageQuantity): Option[String] = value.totalPackage
```

Comment:
This PR is not up to date. Those changes are in the master branch. You shouldn't have those changes here. Something is wrong rebased or merged.

This comment apply to whole file

Reply:
Done